pipelines/shopify_dlt/__init__.py

```diff
@@ -0,0 +1,72 @@
+
+"""Fetches Shopify Orders and Products."""
+
+from typing import Any, Dict, Iterator, Iterator
+
+import shopify
+
+import dlt
+
+from dlt.extract.source import DltResource
+
+
+@dlt.source(name="shopify")
+def shopify_source(
+    private_app_password: str = dlt.secrets.value,
+    api_version: str = dlt.config.value,
+    shop_url: str = dlt.config.value,
```

Comment:
please add one more parameter `start_date` or `start_timestamp` that you can then pass as `initial_value` to `incremental`. we'll have people with millions of orders spanning many years. and maybe they just need to load last year of data

Reply:
I took a slightly different approach, but one that makes more sense imho. lmk what you think.

Reply:
this means that we do not load entities created after `start_date` even if they are updated after `start_date`? it is hard to say if this makes more sense or not... I personally would just filter by `updated_at` but I'm ok with what you did if you explain this in the docstrings below